web-server/v0.4/src/components/PageHeader/index.js

```diff
@@ -193,9 +194,11 @@ export default class PageHeader extends PureComponent {
       tabActiveKey,
       tabDefaultActiveKey,
       tabBarExtraContent,
+      selectedControllers,
     } = this.props;
-    const { breadcrumb } = this.state;
 
+    const { breadcrumb } = this.state;
+    // console.log('nfjewq');
```

Comment:
We have extra `console.log` statements here and on line 117 in `web-server/v0.4/src/pages/ComparisonSelect/index.js`